gtkmm.json

```diff
@@ -0,0 +1,142 @@
+{
```

Comment:
please don't. It's harder to maintain.

Reply:
Ok, I can add it to the original manifest.